Project/Assets/Editor/Annotations/AnnotationInteractionEditor.cs

```diff
@@ -143,4 +188,40 @@ void GetImagePath()
         }
         GUILayout.EndHorizontal();
     }
+
+
+    /// <summary>
+    /// Allows the user to save any edits made to the full annotation to a text file
+    /// </summary>
+    void SaveTextFileButton()
+    {
```

Comment:
If I understand what you are saying, it should already do that.  I added a comment where it does.